test/e2e/test/apmserver/checks_apm.go

```diff
@@ -264,3 +267,53 @@ func countIndex(esClient client.Client, indexName string) (int, error) {
 	}
 	return countResult.Count, nil
 }
+
+const sampleDefaultAgentConfiguration = `{"service":{},"settings":{"transaction_sample_rate":"1","capture_body":"errors","transaction_max_spans":"99"}}`
+
+// CheckAgentConfiguration creates an agent configuration through Kibana and then check that the APM Server is able to retrieve it.
+func (c *apmClusterChecks) CheckAgentConfiguration(apm apmv1.ApmServer, k *test.K8sClient) test.StepList {
+	apmVersion := version.MustParse(apm.Spec.Version)
+
+	if !(apmVersion.IsSameOrAfter(apmv1.ApmAgentConfigurationMinVersion) && apm.Spec.KibanaRef.IsDefined()) {
```

Comment:
isn't the check on `apm.Spec.KibanaRef.IsDefined() ` good enough?

Reply:
Yes, I have been a bit paranoid on this one 😄 Let's remove it and see how it goes.